src/app/onboarding/funnel/steps/LocationStep.tsx

```
@@ -1,17 +1,70 @@
+"use client";
+import { useState, useCallback, useMemo } from "react";
+import type { RegionValue } from "@/app/onboarding/context/regionTypes";
+import React from "react";
+import {
+  StepContainer,
+  StepHeader,
+  StepActions,
+  ActionButton,
+} from "../../components";
+import { Map } from "../../components/Map/Map";
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)